source/agora/network/Manager.d

```diff
@@ -764,6 +764,21 @@ public class NetworkManager
         return this.peers.filter!(p => p.isAuthenticated());
     }
 
+    /***************************************************************************
+
```

Comment:
Needs function and parameter description

Reply:
Updated